operator/controllers/mlops/server_controller.go

```diff
@@ -184,7 +183,7 @@ func (r *ServerReconciler) updateStatus(server *mlopsv1alpha1.Server) error {
 }
 
 // Find Servers that need reconcilliation from a change to a given ServerConfig
-func (r *ServerReconciler) mapServerFromServerConfig(obj client.Object) []reconcile.Request {
+func (r *ServerReconciler) mapServerFromServerConfig(_ context.Context, obj client.Object) []reconcile.Request {
 	logger := log.FromContext(context.Background()).WithName("mapServerFromServerConfig")
```

Comment:
similar context question as above

Reply:
I have added a TODO in the code as per the answer in the previous review comment.